src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -236,11 +236,29 @@ Namespace Microsoft.CodeAnalysis.Semantics
             For Each base In statement.LocalDeclarations
                 If base.Kind = BoundKind.LocalDeclaration Then
                     Dim declaration = DirectCast(base, BoundLocalDeclaration)
-                    builder.Add(OperationFactory.CreateVariableDeclaration(declaration.LocalSymbol, Create(declaration.InitializerOpt), _semanticModel, declaration.Syntax))
+                    Dim initializer As IVariableInitializer = Nothing
+                    If declaration.InitializerOpt IsNot Nothing Then
+                        Debug.Assert(TypeOf declaration.Syntax Is ModifiedIdentifierSyntax)
+                        Dim variableDeclaratorSyntax = DirectCast(declaration.Syntax.Parent, VariableDeclaratorSyntax)
+                        Dim initializerSyntax As SyntaxNode = If(declaration.InitializedByAsNew,
+                            DirectCast(variableDeclaratorSyntax.AsClause, SyntaxNode),
+                            variableDeclaratorSyntax.Initializer)
+                        Dim isImplicit As Boolean = False
+                        If initializerSyntax Is Nothing Then
+                            initializerSyntax = declaration.InitializerOpt.Syntax
+                            isImplicit = True
```

Comment:
Is `isImplicit = True` for `As New ...`? If so, why is that considered implicit?